src/rollbar.php

```diff
@@ -5,6 +5,11 @@
  * Unless you need multiple RollbarNotifier instances in the same project, use this.
  */
 class Rollbar {
+    const LEVEL_ERROR = 'error';
+    const LEVEL_CRITICAL = 'critical';
+    const LEVEL_WARNING = 'warning';
```

Comment:
I recognize it's not in the original, but this is a great opportunity to fix that!